force-app/main/default/classes/AFFL_AccRecordType_TEST.cls

```diff
@@ -115,6 +115,25 @@ public with sharing class AFFL_AccRecordType_TEST {
         insertAfflWAccRecTypeEnforced();
     }
 
+    /* Test to ensure an error is thrown when an affiliation is inserted with an Account whose record
+     * type is not a part of Affiliation Mappings when Affiliation Record Type Enforced is Enabled
+     * for a spanish user.
+     * Test Scenario: T-3939021
```

Comment:
Including the test scenario number here is helpful. I wanted to make a comment to that effect as this is my first time viewing this exact sort of work and wanted to call this out as a useful pattern moving forward.